### ISSUE TITLE
More info in F3 menu

### ISSUE THREAD

**Issue by lFoZardl** [Ashleyz4/MC_b1.7.3-MegaMod#12]
Being able to see the biome and light level

**Comment by ScarletLovell** [comment]
Good idea! This project is sadly currently on hold as I've lost the files. Sorry for the late reply, as I'm just seeing it now. However, I plan to remake the project I just don't have the time for it right now.
